cfg/cppcheck-cfg.rng

```diff
@@ -698,6 +698,7 @@
       <value>push</value>
       <value>pop</value>
       <value>find</value>
+      <value>find-const</value>
```

Comment:
Whats the difference between find and find-const?

Reply:
E.g. `std::string::find()` is always const, `std::set::find()` can be non-const.
`isConstFunctionCall()` checks the action.

Reply:
alright, sounds good.